projects/convai2/baselines/profilememory/eval_f1.py

```diff
@@ -16,6 +16,7 @@
         model='projects.personachat.persona_seq2seq:PersonachatSeqseqAgentSplit',
         model_file='models:convai2/profilememory/profilememory_convai2_model',
         rank_candidates=False,
+        metrics='f1',
```

Comment:
do we need this here since you are setting this at the top level `convai2/eval_f1.py` file?